test/integration/targets/prepare_http_tests/tasks/main.yml

```diff
@@ -1,7 +1,7 @@
 # The docker --link functionality gives us an ENV var we can key off of to see if we have access to
 # the httptester container
 - set_fact:
-    has_httptester: "{{ lookup('env', 'HTTPTESTER') != '' }}"
+    has_httptester: lookup('env', 'HTTPTESTER') != ''
```

Comment:
This may very well be the reason tests are failing.

This no longer templates this and is setting the value to the literal string here.